utils/swift-api-dump.py

```diff
@@ -234,8 +234,7 @@ def collect_frameworks(sdk):
 
 
 def create_dump_module_api_args(cmd_common, cmd_extra_args, sdk, module,
-                                target, source_filename, output_dir, quiet,
-                                verbose):
+                                target, output_dir, quiet, verbose):
```

Comment:
I gripped the entire Swift codebase for use of this function; the only use is about sixty lines down.